lib/Cro/WebSocket/Client.pm6

```diff
@@ -34,10 +34,10 @@ class Cro::WebSocket::Client {
             $!body-serializers = $body-serializers;
         }
 
-        @!headers = (Cro::HTTP::Header.new(name => 'Upgrade', value => 'websocket'),
+        @!headers = [Cro::HTTP::Header.new(name => 'Upgrade', value => 'websocket'),
                      Cro::HTTP::Header.new(name => 'Connection', value => 'Upgrade'),
                      Cro::HTTP::Header.new(name => 'Sec-WebSocket-Version', value => '13'),
-                     Cro::HTTP::Header.new(name => 'Sec-WebSocket-Protocol', value => 'echo-protocol'));
+                     Cro::HTTP::Header.new(name => 'Sec-WebSocket-Protocol', value => 'echo-protocol')];
```

Comment:
You don't need anything around this; the typical way to write such an assignment in Perl 6 is just `@!headers = foo, bar, baz;`.